<launch>
    <node pkg="wifi_publisher" type="wifi_signal_grid_builder" name="wifi_signal_grid_builder" output="screen">
        launch-prefix="xterm -e gdb --args"
        <param name="base_frame" value="/base_link"/>
        <param name="sg_grid" value="/map"/>
        <param name="mapSizeX" value="128" type="int"/>
        <param name="mapSizeY" value="128" type="int"/>
        <param name="resolution" value="1.0"/>
        <param name="maxSignalValue" value="100.0"/>
        <remap from="/WifiSignalGridBuilder/signal" to="/wifiPublisher/wifi_data"/>
        <!--<remap from="/SignalGridBuilder/signalMap" to="/SignalGridBuilder/signalMap"/>-->
    </node>

    <node pkg="wifi_publisher" type="wifi_publisher_node.py" name="wifi_publisher">
        <remap from="wifi_data" to="/wifiPublisher/wifi_data"/>
    </node>

</launch>